pkg/providers/vsphere/validator.go

```diff
@@ -188,51 +158,12 @@ func (v *Validator) ValidateClusterMachineConfigs(ctx context.Context, vsphereCl
 		}
 	}
 
-	if etcdMachineConfig != nil && etcdMachineConfig.Spec.OSFamily != anywherev1.Bottlerocket && etcdMachineConfig.Spec.OSFamily != anywherev1.Ubuntu {
-		return fmt.Errorf("etcd node osFamily: %s is not supported, please use one of the following: %s, %s", etcdMachineConfig.Spec.OSFamily, anywherev1.Bottlerocket, anywherev1.Ubuntu)
-	}
-
-	if etcdMachineConfig != nil && controlPlaneMachineConfig.Spec.OSFamily != etcdMachineConfig.Spec.OSFamily {
-		return errors.New("control plane and etcd machines must have the same osFamily specified")
-	}
-
-	if err := v.validateSSHUsername(controlPlaneMachineConfig); err == nil {
-		for _, wnConfig := range workerNodeGroupMachineConfigs {
-			if err = v.validateSSHUsername(wnConfig); err != nil {
-				return fmt.Errorf("validating SSHUsername for worker node VSphereMachineConfig %v: %v", wnConfig.Name, err)
-			}
-		}
-		if etcdMachineConfig != nil {
-			if err = v.validateSSHUsername(etcdMachineConfig); err != nil {
-				return fmt.Errorf("validating SSHUsername for etcd VSphereMachineConfig %v: %v", etcdMachineConfig.Name, err)
-			}
-		}
-	} else {
-		return fmt.Errorf("validating SSHUsername for control plane VSphereMachineConfig %v: %v", controlPlaneMachineConfig.Name, err)
-	}
-
-	for _, machineConfig := range vsphereClusterSpec.machineConfigsLookup {
-		if machineConfig.Namespace != vsphereClusterSpec.Cluster.Namespace {
-			return errors.New("VSphereMachineConfig and Cluster objects must have the same namespace specified")
```

Comment:
Removing because this is validated [here](https://github.com/aws/eks-anywhere/blob/main/pkg/cluster/vsphere.go#L56)

Reply:
is this link the right one?

Reply:
Right file, wrong line number. Updated it!